.github/workflows/ci.yml

```diff
@@ -45,54 +34,119 @@ jobs:
       - name: Build all packages
         run: npm run build
 
+  get-commands:
+    name: Get scaffold commands
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    runs-on: ubuntu-latest
+    outputs:
+      scaffoldCommands: ${{ steps.set-matrix.outputs.scaffoldCommands }}
+    steps:
+      - id: set-matrix
+        run: echo "scaffoldCommands=[{\"cmd\":\"generate-app\",\"hasBackend\":false},{\"cmd\":\"generate-app-backend\",\"hasBackend\":true},{\"cmd\":\"generate-panel\",\"hasBackend\":false},{\"cmd\":\"generate-datasource\",\"hasBackend\":false},{\"cmd\":\"generate-datasource-backend\",\"hasBackend\":true},{\"cmd\":\"generate-scenes-app\",\"hasBackend\":false}]" >> $GITHUB_OUTPUT
+
+  generate-plugins:
+    name: Test plugin scaffolding
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
maybe it's better to use a label for this? Also, why would you want to skip the CI?